pinner/Makefile.am

```diff
@@ -0,0 +1,25 @@
+include $(top_srcdir)/build/vars.build.mk
+
+plugin = pinner
+
+geanyplugins_LTLIBRARIES = pinner.la
+
+pinner_la_SOURCES = \
+	pinner.c
+
+pinner_la_CPPFLAGS = $(AM_CPPFLAGS) \
+						 -DG_LOG_DOMAIN=\"Pinner\"
+
+pinner_la_CFLAGS = \
+	$(AM_CFLAGS) \
+	-fsanitize=address,undefined
```

Comment:
I enabled the [sanitize option](https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html), and use `LD_PRELOAD=/usr/lib64/libasan.so ./geany -c ~/src/Geany/devconf` to test the plugin. All looks good now that I moved the initialization of the 'container' to a separate function...